src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -180,6 +180,10 @@
     type: int
     short-summary: Load balancer idle timeout in minutes.
     long-summary: Desired idle timeout for load balancer outbound flows, default is 30 minutes. Please specify a value in the range of [4, 100].
+  - name: --load-balancer-backend-pool-type
+    type: string
+    short-summary: Load balancer backend pool type.
+    long-summary: Load balancer backend pool type, supported values are nodeIP and nodeIPConfiguration.
```

Comment:
If the value of this parameter is enumerable, it is suggested to use `arg_type=get_enum_type()` to define the param. After that, it is not necessary to manually specify the available options.

Reply:
Fixed, please take a look.